misc-utils/kill.c

```diff
@@ -68,12 +68,51 @@ enum {
 	KILL_OUTPUT_WIDTH = 72
 };
 
+#if defined (__linux__)
+# include <sys/types.h>
+# ifndef __NR_pidfd_open
+#  include <asm-generic/unistd.h>
+# endif
+# ifndef HAVE_PIDFD_OPEN
+#  if !defined(SYS_pidfd_open) && defined(__NR_pidfd_open)
+#   define SYS_pidfd_open __NR_pidfd_open
```

Comment:
It should be enough to use SYS_ here, and UL_CHECK_SYSCALL() in the configure script. You do not have to play with __NR_ macros here. See for example fallocate.c